test/functional/comment_controller_test.rb

```diff
@@ -258,4 +258,12 @@ def teardown
         type: 'page'
     assert ActionMailer::Base.deliveries.collect(&:subject).include?("New comment on 'Wiki page title'")
   end
+
+  test 'should prompt user if comment includes question mark' do
+    UserSession.create(rusers(:jeff))
+    xhr :post, :create,
+        id: comment.id,
```

Comment:
The tests show the error: 

```

.........................................................E
================================================================================
Error: test_should_prompt_user_if_comment_includes_question_mark(CommentControllerTest): NameError: undefined local variable or method `comment' for #<CommentControllerTest:0x000000000251ad18>
test/functional/comment_controller_test.rb:265:in `block in <class:CommentControllerTest>'
```

The use of `comment` here errored because it's not defined. I /think/ we need to use a node id here... yes -- see: https://github.com/publiclab/plots2/blob/master/app/controllers/comment_controller.rb#L16

So you could use, for example, `node(:blog).id` instead of `comment.id` to use an existing test fixture.